classes/DwcArchiverOccurrence.php

```diff
@@ -414,223 +414,236 @@ public function getExsiccateArr($occid){
 		return $retArr;
 	}
 
-	public function getAssociationStr($occid){
+	public function getAssociationStr($occid, $associationType = null){
+		$occid = filter_var($occid, FILTER_SANITIZE_NUMBER_INT);
```

Comment:
This is deprecated in PHP 8.1.0. You can use `htmlspecialchars()` instead.

Reply:
Crazy. The AI google search results are not to be trusted haha.

My bad, @egbot !

<img width="1552" alt="Screen Shot 2024-01-08 at 11 19 22 AM" src="https://github.com/BioKIC/Symbiota/assets/2775448/1b031ce4-4987-420c-aaad-3056d5db03f6">